StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/VerticalSavedPaymentMethodsViewController.swift

```diff
@@ -178,8 +181,7 @@ class VerticalSavedPaymentMethodsViewController: UIViewController {
     }
 
     private func remove(paymentMethod: STPPaymentMethod) {
-        guard let button = paymentMethodRows.first(where: { $0.paymentMethod.stripeId == paymentMethod.stripeId }),
-                let ephemeralKeySecret = configuration.customer?.ephemeralKeySecret else { return }
```

Comment:
Not blocking: why doesn't SavedPaymentMethodManager do the work of getting the ephemeral key?  Maybe PSVerticalVC should create the Manager with the intent and configuration, let it be responsible for deriving the ephemeral key, and inject it into VerticalSavedPMsVC.

Reply:
Yeah that makes it a bit more testable, refactored so this is the case.